src/client/assets/styles/styles.scss

```diff
@@ -1,3 +1,7 @@
+$icon-font-path: "~bootstrap-sass/assets/fonts/bootstrap/";
+
+@import "~bootstrap-sass/assets/stylesheets/_bootstrap.scss";
```

Comment:
Well then there's no need to include the bundled Bootstrap CSS, plus allows for easier style-overriding in SASS.